.github/workflows/pythonpackage.yml

```diff
@@ -0,0 +1,30 @@
+name: Python package
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```

Comment:
Can we change this to
**on: [push, pull_request]**
so it will run when the current branch is pushed/pulled?